jbmc/src/java_bytecode/character_refine_preprocess.cpp

```diff
@@ -1187,11 +1187,9 @@ exprt character_refine_preprocesst::expr_of_to_chars(
 {
   array_typet array_type=to_array_type(type);
   const typet &char_type=array_type.subtype();
-  array_exprt case1(array_type);
-  array_exprt case2(array_type);
   exprt low_surrogate=expr_of_low_surrogate(chr, char_type);
-  case1.copy_to_operands(low_surrogate);
-  case2.move_to_operands(low_surrogate);
+  array_exprt case1({low_surrogate}, array_type);
+  array_exprt case2({std::move(low_surrogate)}, array_type);
   exprt high_surrogate=expr_of_high_surrogate(chr, char_type);
   case2.move_to_operands(high_surrogate);
```

Comment:
It maybe better to fully construct case2 in one step, and declare both cases const

Reply:
Makes sense, just in a race of whether I get this one or #3691 (getting rid of `move_to_operands`) approved first.